.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1743,37 +1743,291 @@ public static class OpenAIFilesModelFactory {
     }
 }
 namespace OpenAI.FineTuning {
+    public readonly partial struct CreateFineTuningJobRequestModel : IEquatable<CreateFineTuningJobRequestModel> {
```

Comment:
Thus far, I believe we've kept these enumerated types for model choices internal and done representation as plain `string`, with better mechanisms for known model selection an open architecture/design problem to solve. Even if just for consistency, can we do the same here and make these model name extensible enums internal (e.g. via "generatorstubs") and swap their references to strings?

Reply:
luckily no references